### PR TITLE
feat(openrouter): add initial account support

### LINKED ISSUE #945: [Site Support]: openrouter.ai
### Site URL / 站点地址

https://openrouter.ai

### Site operation model / 站点运营性质

Not sure / 不确定

### What failed? / 遇到了什么问题？

自动识别失败，站点返回的数据不符合预期。请手动输入信息，或确保已登录目标站点。

### Failure Type / 失败类型

invalid_response

### Failure Message / 失败提示

自动识别失败，站点返回的数据不符合预期。请手动输入信息，或确保已登录目标站点。

### Were you logged in? / 是否已登录？

Yes / 是

### Capabilities needed / 希望支持的能力

- [ ] Refresh balance or quota / 刷新余额或额度
- [ ] Manage API keys / 管理 API Key
- [ ] Model list / 模型列表
- [ ] Model pricing / 模型价格
- [x] Check-in flow / 签到流程
- [ ] Redeem flow / 兑换流程

### Site type or upstream project / 站点类型或上游项目

_No response_

### Additional context / 其他信息

_No response_

### PR DESCRIPTION
**Pull request by qixing-jk**
## Summary

- add OpenRouter as an account-only provider with Management Key validation, account identity detection, and credit balance refresh
- support browser-assisted Management Key creation from the account dialog, including one-time secret handling and recovery for cancellation, timeout, or uncertain creation results
- integrate OpenRouter accounts with persistence, credential deduplication, localized guidance, analytics, and import/export security notices

## Validation

- `pnpm test:ci`
- `pnpm run validate:push`
- `pnpm run i18n:extract:ci`
- `pnpm run i18n:status`
- Chromium E2E: OpenRouter onboarding and New API rc.22 compatibility scenarios (4/4)
- `pnpm run build:firefox`

## Notes

- OpenRouter support is account-only; Management Keys are stored as account credentials.
- Deterministic browser coverage passed, but a signed-in live OpenRouter smoke test was not run.

## Related issue

Fixes #945

OpenRouter does not provide a documented check-in flow. This PR addresses the actionable site-support scope through account detection, Management Key authentication, and balance refresh.

## Summary by CodeRabbit

* **New Features**
  * Added OpenRouter account support with canonical URL detection, Management Key onboarding, and bootstrap recovery UI.
  * Added temp-window Management Key creation orchestration, including authenticated vs logged-out auto-detect behavior.
  * Added exact-credential de-duplication UI with “same management key” labeling.
* **Bug Fixes**
  * Improved cancellation, timeout, and idempotency handling with stronger defensive validation and result normalization.
  * Improved privacy controls to avoid leaking sensitive details in UI responses, logs, and analytics.
* **Documentation**
  * Added OpenRouter foundation design and implementation plan docs.
* **Tests**
  * Added extensive unit, e2e, and contract coverage for OpenRouter flows and edge cases.
* **Chores**
  * Updated the WXT dev dependency.

### CODE REVIEW

File: src/entrypoints/content/messageHandlers/openrouter/managementKeyPage.ts [hunk 1]
```diff
@@ -0,0 +1,485 @@
...
+        if (
+          !isVisible(dialog) ||
+          (dialog !== createDialog && dialogVisibilityBeforeCreate.has(dialog))
+        )
+          return []
+        const codes = Array.from(dialog.querySelectorAll("code"))
+        if (codes.length !== 1 || !isVisible(codes[0])) return []
+        const secret = normalizeOpenRouterManagementKeySecret(text(codes[0]))
+        return secret ? [{ dialog, secret }] : []
+      })
```
[the remark this answers is not in the document]
Comment (qixing-jk) [reply src/entrypoints/content/messageHandlers/openrouter/managementKeyPage.ts:443]: Keeping this fail-closed intentionally. A pre-mounted hidden dialog can already contain a stale valid secret; accepting it solely because it becomes visible cannot prove this click created that key. The `/key` validation proves credential validity, not creation causality, so this could save an old key while leaving the newly created remote key orphaned. Existing regression coverage preserves the stale-hidden-dialog rejection. Supporting pre-mounted dialogs safely requires evidence of a post-click DOM or secret-content transition from the captured baseline, which is a separate contract change.
